index.js

```diff
@@ -1,22 +1,66 @@
 const axios = require('axios');
 
-const apiUrl = 'https://jsonplaceholder.typicode.com';
+const Url = 'https://jsonplaceholder.typicode.com/';
 
-const getPosts = async () => {
+const getUser = () => {
+  return axios.get(Url + 'users').then(res => {return res.data}).catch(err=>{return err});
+}
+
+const getToDo = () => {
+  return axios.get(Url + 'todos').then(res => {return res.data}).catch(err=>{return err});
+}
+
+const getPost = () => {
+  return axios.get(Url + 'posts').then(res => {return res.data}).catch(err=>{return err});
+}
+
+const main = async () => {
   try {
-    const { data: todos } = await axios.get(`${apiUrl}/todos`);
-    todos.forEach( async (todo) => {
+    const users = await getUser();
+    const todos = await getToDo();
+    const posts = await getPost();
+
+    const userToDo = [0,0,0,0,0,0,0,0,0,0]; 
+    // array userToDo untuk menyimpan data banyaknya todo yang belum diselesaikan user, 
+    // elemen ke-i menunjukan banyaknya ToDo yang belum diselesaikan oleh user dengan id i+1 
+
+    const firstTwoPost = [[0,0,0],[0,0,0],[0,0,0],[0,0,0],
+                          [0,0,0],[0,0,0],[0,0,0],[0,0,0],
+                          [0,0,0],[0,0,0]];
```

Comment:
lebih baik inisiasi variabel jangan di hard code, tapi pake kode aja, misal pake for loop